blivet/devicetree.py

```diff
@@ -144,6 +141,11 @@ def devices(self):
 
         return devices
 
+    @property
+    def names(self):
+        """ List of devices names currently in the tree """
+        return [dev.name for dev in self.devices]
```

Comment:
I think this needs to be something like `[dev.name for dev in self._devices + self._hidden]` to account for hidden devices. I wonder if it also needs some special list of names of LVM logical volumes that are part of incomplete/unusable volume groups to make sure we do not create new ones with in-use names.

Reply:
Fixed.